### PR TITLE
Ecommerce PyCharm integration notes

### PR DESCRIPTION
**Pull request by staubina**
Small Docs update for notes on setting up Pycharm Integration with ecommerce.

### CODE REVIEW

File: docs/pycharm_integration.rst [hunk 1]
```diff
@@ -101,6 +115,16 @@ be set to the service-specific port from the table above.
 *Note*: See next section for additional changes needed for LMS and
 Studio.
 
+Setup a Server Run/Debug Configuration for ecommerce
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add the following Environment Variables to the Server configuration, these are all
+defined on the ecommerce server in /edx/app/ecommerce/ecommerce_env
+
+export PATH="/edx/app/ecommerce/nodeenvs/ecommerce/bin:/edx/app/ecommerce/venvs/ecommerce/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"
```
Comment (clintonb) [inline docs/pycharm_integration.rst]: Write these in a manner that can easily be pasted into PyCharm. Copying the block below, and clicking PyCharm's paste option will create two environment variables for you.

```
ECOMMERCE_CFG=/edx/etc/ecommerce.yml
DJANGO_SETTINGS_MODULE=ecommerce.settings.devstack
```

Comment (clintonb) [reply docs/pycharm_integration.rst]: Have you tried running without setting the PATH envvar? I can't recall ever setting, and don't think it's needed.

**Review by clintonb — APPROVED** [review]

File: docs/pycharm_integration.rst (hunk 1 above)
Comment (staubina) [reply docs/pycharm_integration.rst]: I can try that before merging.

**Comment by robrap** [comment]
@staubina I don't plan on reviewing.  I imagine @clintonb's review was enough, unless you still want others to review as well.

**Comment by staubina** [comment]
@robrap I think I am all set, just need to take a second away from other work and update/merge.

**Pull request merged**